openlibrary/core/vendors.py

```diff
@@ -187,6 +206,16 @@ def clean_amazon_metadata_for_load(metadata):
     if metadata.get('source_records'):
         asin = metadata.get('source_records')[0].replace('amazon:', '')
         conforming_metadata['identifiers'] = {'amazon': [asin]}
+    title, subtitle = split_amazon_title(metadata['title'])
+    conforming_metadata['title'] == title
+    if subtitle:
+        conforming_metadata['full_title'] = title + ' : ' + subtitle
+        conforming_metadata['title'] = title
+        conforming_metadata['subtitle'] = subtitle
+    # Record original title if some content has been removed (i.e. parentheses)
+    if metadata['title'] != conforming_metadata.get('full_title', conforming_metadata['title']):
+        conforming_metadata['notes'] = "Source title: %s" % metadata['title']
```

Comment:
perhaps only set this if `( )` were present in the original

Reply:
I think I like your current `if`-check better; checking for `(` makes this code rely on the internals of `split_amazon_title`. Just checking if it's different has better decoupling (and is easier to read).

Reply:
ok, you have convinced me :)